modules/oc-adm-by-example-content.adoc

```diff
@@ -346,6 +350,96 @@ Display and filter node logs
 
 
 
+== oc adm ocp-certificates monitor-certificates
+Watch platform certificates.
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Watch platform certificates.
+  oc adm ocp-certificates monitor-certificates
+----
+
+
+
+== oc adm ocp-certificates regenerate-leaf
+Regenerate client and serving certificates of an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate all known leaf certificates on the cluster.
+  oc adm certificates regenerate-leaf -A --all
+  
+  # Regenerate a leaf certificate contained in a particular secret.
+  oc adm ocp-certificates regenerate-leaf -n openshift-config-managed kube-controller-manager-client-cert
+----
+
+
+
+== oc adm ocp-certificates regenerate-machine-config-server-serving-cert
+Regenerate the machine config operator certificates in an OpenShift cluster
+
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate the MCO certs without modifying user-data secrets
+  oc adm certificates regenerate-machine-config-server-serving-cert --update-ignition=false
```

Comment:
command failed:
`oc adm certificates regenerate-machine-config-server-serving-cert --update-ignition=false
error: unknown flag: --update-ignition
See 'oc adm --help' for usage.`

Reply:
Removing this example temporarily